package.json

```diff
@@ -78,10 +83,12 @@
     "eslint-plugin-n": "^15.7.0",
     "eslint-plugin-qunit": "^7.3.4",
     "eslint-plugin-unicorn": "^46.0.0",
+    "jest": "^29.5.0",
```

Comment:
I tried vitest first but the types in the current version are whack and I got impatient waiting for them to fix it.

Reply:
Interesting! You found the types in `jest` to be better than `vitest`?

Reply:
Not really. The latest `vitest` version had a bunch of type bugs in it (I found related issues but don't remember what it was now) and rather than wait for them to fix and ship, I just switched to `jest` since the API is basically the same. Might be fixed now. I just put this comment here bc I knew you were going to suggest trying `vitest`. :-P